adds impeachment polls dataset to fivethirtyeight package, 3rd attempt

fatimak98 opened the pull request:
Hey @rudeboybert! 

Thanks for your feedback - sorry for all the excess files, not sure why that happened. I followed the steps you left in my last PR attempt. Hopefully this PR works, fingers crossed! Tagging @beanumber

fatimak98 committed:
adds impeachment polls dataset to fivethirtyeight package, 3rd attempt

fatimak98 committed:
adds impeachment polls dataset to fivethirtyeight package, 3rd attempt

rudeboybert commented on R/data_fatima.R:
```diff
@@ -0,0 +1,35 @@
+#'   \item{text}{poll question}
+#'   \item{category}{category of poll question with 5 categories: impeach and remove, begin proceedings, begin inquiry, reasons, impeach}
+#'   \item{include}{yes/no logical -- value unknown}
+#'   \item{yes}{% of repsondents in sample who answered "Yes" to the poll question}
```
Unfortunately it seems all the following `%` signs are causing errors. Could you replace them all with the word `Percent`?

fatimak98 replied:
Hi Albert! Sorry for the delay on this, I made the requested change above. Let me know if there are any other changes I need to make.

rudeboybert replied:
Eep! It seems a merge conflict has creeped in as evidenced by the 
```
<<<<<<< HEAD
```
and the
```
=======
```
and the
```
>>>>>>> 25e8defc63e3a707dcecaa074932ec91ea22f474
```
Could you remove these lines, and remove the redundant entries? Almost there!

fatimak98 replied:
I've removed the lines and redundant entries (hopefully), please let me know if you can see these changes on your end?

rudeboybert reviewed (commented):
Hey Fatima, you're almost done! One more change is needed. After making this change, don't forget to build and check your package before committing; this tests that the package is suitable for submission to CRAN.

fatimak98 committed:
changed % to Percent

beanumber commented:
@fatimak98 please attend to these changes so we can close the book on this. Thanks!

fatimak98 committed:
changed '%' to 'Percent' in data.R file for impeachment polls dataset

Merge branch 'adds-impeachment-polls' of https://github.com/fatimak98/fivethirtyeight into adds-impeachment-polls

# Conflicts:
#	R/data_fatima.R

fatimak98 committed:
updated data_Fatima.R file, got rid of merge conflicts

rudeboybert reviewed (approved):
Looks good!

rudeboybert commented:
Great! You're done! You're now listed as a package [contributor](https://github.com/rudeboybert/fivethirtyeight/graphs/contributors). Thanks for your work @fatimak98!

fatimak98 commented:
Thank you so much @rudeboybert !
